src/routes/Gavin/modules/Variants.js

```diff
@@ -0,0 +1,69 @@
+// ------------------------------------
+// Constants
+// ------------------------------------
+export const SET_VARIANTS = 'Gavin.SET_VARIANTS'
+
+export const constants = { SET_VARIANTS }
+
+// ------------------------------------
+// Action creators
+// ------------------------------------
+export function setVariants (variants) {
+  return {
+    type    : SET_VARIANTS,
+    payload : variants
+  }
+}
+
+export const actions = { setVariants }
+
+// ------------------------------------
+// Action Handlers
+//
+// Switch that defines what every action
+// should do
+// ------------------------------------
+const ACTION_HANDLERS = {
+  [SET_VARIANTS] : (state, action) => {
```

Comment:
I think the name would be more event-y if it's something like LOADED_VARIANTS or so.